src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -720,10 +769,14 @@ cdef class WidomInsertion(ReactionAlgorithm):
 
     def measure_excess_chemical_potential(self, reaction_id=0):
         """
-        Measures the excess chemical potential in a homogeneous system.
+        Measures the excess chemical potential in a homogeneous system for the provided reaction_id.
+        Please register the insertion moves first via calling the method add_reaction (with only product types specified).
```

Comment:
It it not clear in the given context what it means to "register the insertion move". I propose to rephrase this as: "The reaction with given reaction_id has to be defined before."

Reply:
You define a certain insertion move (e.g. emptyset -> 1 A, for inserting one particle of type A, or emptyset -> 1 A+1 B for inserting one particle of type A and one particle of type B, via calling add_reaction() previously). That is why there is a reference to add_reaction.

Reply:
As mentioned before
> It is not clear in the given context.

Please, explain it to the reader, not to me.

One should keep the terminology consistent. If the documentation of the reaction ensemble uses the term _define_ a reaction and not _register_, then you should use the same term here.